### PR TITLE
#312 - Change "Jobs Board" to "Jobs"

### PR DESCRIPTION
**Pull request by dfcook**
As part of fixing this I also fixed the console errors in the job view.

Plus, I moved the webpack config to version 4 which dramatically simplifies it, tested in both dev and prod configurations.

### CODE REVIEW

**Comment by dfcook** [comment]
Added fix for 239, this change includes:

1. Implementing a global plugin for authorization which adds the bearer token to all axios requrests when authenticated.
2. Remove all manual token adds from store actions (other than jobs apply as it passes form data).
3. In the plugin login fetch profile data, and clear it on logout.
4. Change LoginView to call the $auth login instead of the store.
5. At point of app creation load profile data if the token is in the store.

**Comment by dfcook** [comment]
#172 Also added using vue-toasted to alert errors rather than the alert box.

I've also changed how we source external asses. Bootstrap, font-awesome, material icons and jquery are now all handled using npm/webpack.

This makes it easier for us to ensure we have up-to-date packages and also to customize our bootstrap using scss when it comes to re-design

File: package.json [hunk 1]
```diff
@@ -25,64 +25,66 @@
...
-    "babel-plugin-transform-flow-strip-types": "^6.22.0",
     "babel-plugin-transform-runtime": "^6.23.0",
     "babel-preset-env": "^1.6.1",
     "babel-preset-stage-2": "^6.24.1",
     "babel-register": "^6.26.0",
     "chai": "^4.1.2",
-    "chalk": "^2.3.0",
-    "chromedriver": "^2.35.0",
+    "chalk": "^2.3.2",
+    "chromedriver": "^2.36.0",
```
Comment (jasonify) [inline package.json]: I know I probably added this but we might move this to dev-dependencies

**Comment by jasonify** [comment]
Hey great work :). I just started testing it but seeing that Edit Profile seems to not be loading?


Ideally we could have a PR like this one that touches so many key components into a few smaller PRs, I could see this being 3-4 PRs (toast, auth, webpack upgrade).  It would be less scary if we had some more end-to-end tests but right now there's a lot of manual testing. 

I'm thinking that at the very least we should have a "manual checklist" we should run through i.e register, login, logout, edit profile, create job, subscribe, play podcast, etc. It will be annoying but at least we can be a little more confident we won't forget to check key components.

**Comment by dfcook** [comment]
I've fixed EditProfileView.

I agree, this should be multiple PR's, the fact it isn't is down to me not really being 100% confident with git having used other SCM's such as subversion or TFS before now. I tend to start working on my fork, raise a PR once I've got some fixes then start working on a new fix. I need to learn how to work with branches effectively to split these up.

Do you want me to cancel this and raise separate PR's?

**Comment by jasonify** [comment]
Let me test this a bit more and I'll let you know.

I understand the problem, I am trying to get better at it myself.

**Comment by jasonify** [comment]
Now that I think about it a bit more. I think it would be great if you could just submit the the auth + toast as their own PR and the webpack / build stuff as another PR. 

I am going to prioritize the testing of the auth + toast since that is most likely to have merge conflicts.


The webpack stuff I would rather have time to test and not rush. 

Thanks

**Comment by dfcook** [comment]
Closing this as raising separate PR's

**Pull request closed without merging**